UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Model/GameServerConnectionInfo.cs

```diff
@@ -1,17 +1,32 @@
-﻿namespace PlayFab.MultiplayerAgent.Model
+namespace PlayFab.MultiplayerAgent.Model
 {
+    using System;
     using System.Collections.Generic;
     using Json;
 
+    /// <summary>
+    /// A class that captures details on how a game server operates.
+    /// </summary>
     public class GameServerConnectionInfo
     {
         public GameServerConnectionInfo()
         {
         }
 
+        /// <summary>
+        /// The IPv4 address of the game server.
+        /// </summary>
+        [JsonProperty(PropertyName = "publicIpV4Address")]
+        public string PublicIPv4Address { get; set; }
+
+
+        [Obsolete("Please use PublicIPv4Address instead.")]
         [JsonProperty(PropertyName = "publicIpV4Adress")]
-        public string PublicIpV4Adress { get; set; }
+        public string PublicIpV4Adress { get => PublicIPv4Address; set { if (!string.IsNullOrWhiteSpace(value) && PublicIPv4Address != value) { PublicIPv4Address = value; } } }
```

Comment:
The validation is good, but I'm afraid of the confusion this will bring, since the properties differ only on the 'v'. I would skip that, since an invalid IP address will throw eventually.